api/main_events/migrations/0001_initial.py

```diff
@@ -1,4 +1,4 @@
-# Generated by Django 2.0.5 on 2019-02-07 04:43
```

Comment:
Did you mean to commit this?

Reply:
Oops, that shouldn't have been committed

Reply:
Can you remove it from this PR?

Reply:
Done